dist/utils.min.js

```diff
@@ -1,2 +1,2 @@
+"use strict";angular.module("ts.utils",[]),angular.module("ts.utils").directive("uiEvent",function($parse){return{priority:100,link:function($scope,$elm,$attrs){var events=$scope.$eval($attrs.uiEvent);angular.forEach(events,function(uiEvent,eventName){var fn=$parse(uiEvent);$elm.bind(eventName,function(evt){var params=Array.prototype.slice.call(arguments);params=params.splice(1),fn($scope,{$event:evt,$params:params}),$scope.$$phase||$scope.$apply()})})}}}),angular.module("ts.utils").filter("tsTruncate",function(){return function(value,wordLimit){var ellipses=arguments.length<=2||void 0===arguments[2]?"…":arguments[2];if(!value||!angular.isNumber(wordLimit))return value;var words=value.split(" ");return words.length>wordLimit&&(value=words.slice(0,wordLimit).join(" "),ellipses&&(value+=ellipses)),value}}),angular.module("ts.utils").directive("scrollOn",function($timeout){return{link:{pre:function($scope,$element,$attrs){$scope.$on($attrs.scrollOn,function(event,location){$timeout(function(){$element[0].scrollTop="bottom"===location?$element[0].scrollHeight:"top"===location?0:location},!0)})}}}}),angular.module("ts.utils").directive("focusOn",function(){return{link:function($scope,$element,$attrs){var listener=angular.noop;$attrs.$observe("focusOn",function(newVal){listener(),listener=$scope.$on(newVal,function(){$element[0].focus()})})}}}),angular.module("ts.utils").directive("autoGrow",function($timeout){return{restrict:"A",link:function($scope,$element,$attrs){function grow(){$element[0].style.height=0,$element[0].style.height=$element[0].scrollHeight+"px"}$element.on("input",grow),$timeout(grow,!0)}}});
```

Comment:
Line is too long.<br>Use the function form of "use strict".<br>Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>Expected '{' and instead saw 'return'.<br>'$attrs' is defined but never used.